apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/OpenAPIContentProvider.java

```diff
@@ -0,0 +1,220 @@
+  // strings. So we'll fal back on tried-and-true structure to avoid
+  // unpleasantness.
+
+  /** Open API content short description for /Patient resource bundle offset. */
+  public static final String PATIENT_SP_RES_ID_SHORT =
+      "The patient resource identifier (_id) to search for";
+  /** Open API content value for /Patient resource bundle offset. */
+  public static final String PATIENT_SP_RES_ID_VALUE =
+      "Fetch <i>Patient<i/> data using a FHIR <i>IdType<i/> identifier; an IdType"
+          + " represents the logical identity for a resource, or as much of that"
+          + " identity that is known. In FHIR, every resource must have a <i>logical ID<i/> which is"
+          + " defined by the FHIR specification as:"
+          + "<p><code>"
+          + "Any combination of upper or lower case ASCII letters ('A'..'Z', and 'a'..'z', numerals ('0'..'9'),"
+          + " '-' and '.', with a length limit of 64 characters. (This might be an integer, an un-prefixed OID, UUID"
+          + " or any other identifier pattern that meets these constraints.)</code></p><p>"
+          + "This class contains that logical ID, and can optionally also contain a relative or absolute URL"
+          + " representing the resource identity; the following are all valid values for IdType, and all might"
+          + " represent the same resource:</p><ul>"
+          + "<li><code>123</code> (just a resource\u0027s ID)</li>"
+          + "<li><code>Patient/123</code> (a relative identity)</li>"
+          + "<li><code>http://example.com/Patient/123 (an absolute identity)</code></li>";
+
+  /** Open API content short description for /Patient resource bundle offset. */
+  public static final String PATIENT_START_INDEX_SHORT =
+      "The starting offset used for result pagination";
+  /** Open API content value for /Patient resource bundle offset. */
+  public static final String PATIENT_START_INDEX_VALUE =
+      "When fetching a <i>Bundle Response</i> using pagination, this URL parameter represents an offset"
+          + " (starting point) into the list of elements for the <i>Request</i>."
+          + "<p/>Example:<ul>"
+          + "<li>startIndex=100</li>";
+
+  /** Open API content short description for /Patient resource bundle offset. */
+  public static final String PATIENT_LAST_UPDATED_SHORT = "When the resource version last changed";
+  /** Open API content value for /Patient resource bundle offset. */
+  public static final String PATIENT_LAST_UPDATED_VALUE =
+      "Only satisfy the Search if the Beneficiary\u0027s <i>last_updated</i> Date"
+          + " falls within a specified <i>DateRange<i/>. A DateRange can include both"
+          + " lo and hi date values, only a lo date value, or only a hi date value."
+          + "<p/>Examples:<ul>"
+          + "<li>&_lastUpdated=gt2023-01-02&_lastUpdated=lt2023-05-01</li>"
+          + "<li>&_lastUpdated=gt2023-01-02</li>"
+          + "<li>&_lastUpdated=lt2023-05-01</li></ul><p/>";
```

Comment:
From what I remember markdown is tricky. Will bring it up in 11th minutes.

Reply:
> From what I remember markdown is tricky. Will bring it up in 11th minutes.

I don't think @mackec1 had a **strong** objection to using markdown, but he did mention that it might not be as helpful as we think–have you tried it?